### PR TITLE
UXG-1307 standalone settings feedback

### PR DESCRIPTION
**Pull request by nicmarson**
http://jira.hq.idx.org/browse/UXG-1307

### CODE REVIEW

File: src/vue/backend/src/mixins/pageGuard.js [hunk 1]
```diff
@@ -25,9 +25,25 @@ export default {
                 this.$store.dispatch(`${this.module}/setItem`, { key, value: changes[key] })
             }
         },
+        scrollToTop () {
+            console.log('scroll')
```
Comment (sgreenl5) [inline src/vue/backend/src/mixins/pageGuard.js:29]: Test code

File: src/vue/backend/src/views/children/advancedContentTab.vue [hunk 2]
```diff
@@ -43,6 +43,9 @@ export default {
             this.formDisabled = false
             if (status === 200) {
                 this.saveAction()
+            } else {
+                // To do: user feedback
```
Comment (sgreenl5) [inline src/vue/backend/src/views/children/advancedContentTab.vue:47]: You can remove the comment here

File: src/vue/backend/src/views/children/generalContentTab.vue [hunk 3]
```diff
@@ -43,7 +43,9 @@ export default {
             this.formDisabled = false
             if (status === 204) {
                 this.saveAction()
-                // To Do: User feed back
+            } else {
+                // To do: user feedback
```
Comment (sgreenl5) [inline src/vue/backend/src/views/children/generalContentTab.vue:47]: You can remove the comment

File: src/vue/backend/src/views/children/idxContentTab.vue [hunk 4]
```diff
@@ -44,7 +44,9 @@ export default {
             this.formDisabled = false
             if (status === 204) {
                 this.saveAction()
-                // To Do: User feed back
+            } else {
+                // To do: user feedback
```
Comment (sgreenl5) [inline src/vue/backend/src/views/children/idxContentTab.vue:48]: You can remove the comment

File: src/vue/backend/src/views/settings/Listings.vue [hunk 5]
```diff
@@ -56,6 +56,9 @@ export default {
             this.formDisabled = false
             if (status === (204 || 200)) {
                 location.reload()
+            } else {
+                // To do: user feedback
```
Comment (sgreenl5) [inline src/vue/backend/src/views/settings/Listings.vue:60]: You can remove the comment

File: src/vue/backend/src/views/settings/Omnibar.vue [hunk 6]
```diff
@@ -40,7 +40,16 @@ export default {
...
     methods: {
         saveHandler () {
-            // To Do: api connection
+            this.formDisabled = true
+            if (this.formChanges) {
+                this.formDisabled = false
+                if (status === 204) {
+                    this.saveAction()
+                } else {
+                    // To do: user feedback
```
Comment (sgreenl5) [inline src/vue/backend/src/views/settings/Omnibar.vue:49]: Comment

File: src/vue/backend/src/views/settings/SocialPro.vue [hunk 7]
```diff
@@ -64,7 +64,19 @@ export default {
...
-        })
+        }),
+        saveHandler () {
+            this.formDisabled = true
+            if (this.formChanges) {
+                this.formDisabled = false
+                if (status === 204) {
+                    this.saveAction()
+                } else {
+                    // To do: user feedback
```
Comment (sgreenl5) [inline src/vue/backend/src/views/settings/SocialPro.vue:75]: Comment

File: src/vue/backend/src/mixins/pageGuard.js [hunk 8]
```diff
@@ -25,9 +25,25 @@ export default {
...
+            window.scrollTo(0, 0)
```
Comment (sgreenl5) [inline src/vue/backend/src/mixins/pageGuard.js:30]: This is a little jumpy, lets add smooth behavior

**Review by sgreenl5 — CHANGES_REQUESTED** [review]
Looks good! Remove some of those comments, and add the scroll behavior, and this is set! Great job 👍

File: src/vue/backend/src/components/Notification.vue [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+    props: {
+        notification: {
+            type: Object,
+            default: () => {}
+        }
+    }
+}
+</script>
+<style lang="scss">
+.notification {
```
Comment (sgreenl5) [inline src/vue/backend/src/components/Notification.vue:29]: This is missing the transition css to make it fade in and out

**Review by sgreenl5 — CHANGES_REQUESTED** [review]

**Review by sgreenl5 — APPROVED** [review]

**Review by ErikIDX — APPROVED** [review]

**Pull request merged**